pyrogram/client/filters/filters.py

```diff
@@ -89,13 +89,13 @@ class Filters:
     venue = build("Venue", lambda _, m: bool(m.venue))
     """Filter messages that contain :obj:`Venue <pyrogram.api.types.pyrogram.Venue>` objects."""
 
-    private = build("Private", lambda _, m: bool(m.chat.type == "private"))
+    private = build("Private", lambda _, m: bool(m.chat and m.chat.type == "private"))
```

Comment:
Good, extra checks here won't do any harm. We still need to find a way to make private, group and channel filters work for deleted messages, though